# Patch to be able to handle a missing GLFW_MOUSE_PASSTHROUGH definition:
#
# https://github.com/raysan5/raylib/commit/2ad7967db80644a25ca123536cf2f6efcb869684

--- ./src/rcore.c	2022-08-13 16:42:38.000000000 -0400
+++ ./src/rcore.c	2022-08-18 01:01:25.000000000 -0400
@@ -237,6 +237,12 @@
         //#define GLFW_EXPOSE_NATIVE_COCOA    // WARNING: Fails due to type redefinition
         #include "GLFW/glfw3native.h"       // Required for: glfwGetCocoaWindow()
     #endif
+    
+    // TODO: HACK: Added flag if not provided by GLFW when using external library
+    // Latest GLFW release (GLFW 3.3.8) does not implement this flag, it was added for 3.4.0-dev
+    #if !defined(GLFW_MOUSE_PASSTHROUGH)
+        #define GLFW_MOUSE_PASSTHROUGH      0x0002000D
+    #endif
 #endif
 
 #if defined(PLATFORM_ANDROID)
